<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>encryption</class>
 <widget class="QWidget" name="encryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>encryption</string>
  </property>
  <widget class="QStackedWidget" name="encryption_pages">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>760</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>850</width>
     <height>600</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="encryption_open_dir_page">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>2</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>760</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QPushButton" name="encryption_open_dir_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>550</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>760</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>850</width>
       <height>600</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>导入文件夹</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="encryption_para_page">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>2</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>760</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QPushButton" name="encryption_output_path_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择输出文件</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="encryption_input_entry">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>600</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="encryption_input_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="encryption_import_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>已导入文件:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="encryption_process_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始处理</string>
     </property>
    </widget>
    <widget class="QLabel" name="encryption_output_path_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>800</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 247, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QListView" name="encryption_import_list_view">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>800</width>
       <height>180</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 247, 247);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
